code/game/machinery/_machines_base/machinery.dm

```diff
@@ -429,9 +429,9 @@ Class Procs:
 	if(. && !CanFluidPass())
 		fluid_update()
 
-/obj/machinery/get_cell()
+/obj/machinery/get_cell(var/functional_only)
```

Comment:
Should consider making `TRUE` the default here. Also, please make this and `terminal` use the same arg name.

Reply:
Did both